src/scenarios/shared/runner.py

```diff
@@ -85,7 +94,7 @@ def run(self):
                              timeout=self.traits.timeout,
                              warmup='true',
                              iterations=self.traits.iterations,
-                             scenariotypename='%s (%s)' % (const.SCENARIO_NAMES[const.SDK], 'Clean Build'),
+                             scenariotypename='%s_%s' % (const.SCENARIO_NAMES[const.SDK], const.CLEAN_BUILD),
```

Comment:
this is going to be 'clean_build', not 'Clean Build'. Which do you intend?

Reply:
clean_build bc we want to remove the space in filenames...

Reply:
I think we should remove spaces more generically (in our upload function) and leave the scenario names as we want them to be. This string also is what is used on the webpage for display - I'd rather those are human readable. What do you think?

Reply:
Sounds good to me! Could make change in the following PR.

Reply:
That's fine but FYI - test names are their identity. So changing this later will break the trend. It's fine to merge this so that we can make sure everything works as we want it to. We'll basically get new trends when we make the next change. This is one of the few real invariants in our system so we need to be careful around it.

Reply:
Actually removing the spaces won't thoroughly fix the problem. The artifacts' url is not valid bc it contains '\' which is not accepted by helix. Still keeping in touch with the engineering services folks...